cmd/storj/cmd/cp.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/minio/minio/pkg/hash"
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/cfgstruct"
+	"storj.io/storj/pkg/process"
+)
+
+var (
+	cpCfg Config
+	cpCmd = &cobra.Command{
+		Use:   "cp",
+		Short: "A brief description of your command",
+		RunE:  copy,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(cpCmd)
+	cfgstruct.Bind(cpCmd.Flags(), &cpCfg, cfgstruct.ConfDir(defaultConfDir))
+	cpCmd.Flags().String("config", filepath.Join(defaultConfDir, "config.yaml"), "path to configuration")
+}
+
+func copy(cmd *cobra.Command, args []string) (err error) {
```

Comment:
does copy work both ways? what happens if the second argument is local and the first argument is remote?

Reply:
It looks like in aws s3 if the object comes before the local file, it thinks you're downloading. Even if the object doesn't exist. For instance, I input this:

aws s3 --endpoint=http://localhost:7777/ cp s3://myBucket/test123.jpg ~/Desktop/chick3.jpeg

and got this:

download: s3://myBucket/test123.jpg to ../../../../../../Desktop/chick3.jpeg

Reply:
Oh, never mind. I see what you're saying. Working on it